.github/workflows/main.yml

```diff
@@ -17,6 +17,8 @@ jobs:
           secrets: |
               SECRET_KEY
               SLACK_SIGNING_SECRET
+              SLACK_WEBHOOK_BOT_STATUS 
         env:
             SECRET_KEY: ${{ secrets.SECRET_KEY }}
-            SLACK_SIGNING_SECRET: ${{ secrets.SLACK_SIGNING_SECRET }}
\ No newline at end of file
+            SLACK_SIGNING_SECRET: ${{ secrets.SLACK_SIGNING_SECRET }}
+            SLACK_WEBHOOK_BOT_STATUS: ${{ secrets.SLACK_WEBHOOK_BOT_STATUS }}
```

Comment:
add a newline too it's good practice

Reply:
At the end of the file? Does the new commit address it?

Reply:
yup it addresses it, I wanted it bc nisala's pr adds a newline too so it would have conflicted in one of the places